spec/order_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'order'
+
+describe Order do
+  subject(:menu) { described_class.new }
+  subject(:order) { described_class.new }
+  let(:dish) { :dish }
+  let(:price) { 5 }
```

Comment:
Would the test still work if   let(:price) { price } instead of a fixed example?